homeassistant/components/freebox/router.py

```diff
@@ -72,7 +72,9 @@ async def get_hosts_list_if_supported(
     supports_hosts: bool = True
     fbx_devices: list[dict[str, Any]] = []
     try:
-        fbx_devices = await fbx_api.lan.get_hosts_list() or []
+        fbx_devices_pub = await fbx_api.lan.get_hosts_list("pub") or []
+        fbx_devices_wifiguest = await fbx_api.lan.get_hosts_list("wifiguest") or []
```

Comment:
Is `wifiguest` always correct?
Should we set some flag so we know which device trackers are guests?

Reply:
There is no way to configure this value on the Freebox OS.
I got `wifiguest` value in the result of the following API request: `GET /api/v4/lan/browser/interfaces/` ([source](https://dev.freebox.fr/sdk/os/lan/#))

Good point for the flag. But I don't know the best design practices for defining this flag in a device tracker.

Reply:
Just implement an `extra_state_attributes` property in the device tracker entity.
The `unifi` integration actually has an `is_guest` flag:

https://github.com/home-assistant/core/blob/e2dd88be6ef0792b764b0bb39374ffe0b0d511eb/homeassistant/components/unifi/device_tracker.py#L42-L64

https://github.com/home-assistant/core/blob/e2dd88be6ef0792b764b0bb39374ffe0b0d511eb/homeassistant/components/unifi/device_tracker.py#L349-L362